test/App/Fossa/FossaAPIV1Spec.hs

```diff
@@ -0,0 +1,54 @@
+module App.Fossa.FossaAPIV1Spec (spec) where
+
+import App.Fossa.FossaAPIV1 (Issue (..), IssueRule (..), IssueType (..), Issues (..))
+import Data.Aeson (FromJSON, ToJSON, fromJSON, toJSON)
+import Data.Text (Text)
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Test.Hspec
+import Test.Hspec.Hedgehog
+import Prelude
+
+spec :: Spec
+spec = do
+  describe "Issues ToJSON/FromJSON instances" $ do
+    it "are roundtrippable" $ hedgehog $ do
+      issues <- forAll genIssues
+      roundtripJson issues
+
+genIssues :: Gen Issues
+genIssues =
+  Issues
+    <$> Gen.int (Range.linear minBound maxBound)
+    <*> Gen.list (Range.linear 0 100) genIssue
+    <*> arbitraryText
+
+genIssue :: Gen Issue
+genIssue =
```

Comment:
These are annoying to write -- there's lots of boilerplate. Upside: the test was two lines of code with the boilerplate out of the way